File: Solutions/JavaScript/pattern005.js
// Pattern Programs - 005 [JavaScript]
// Github Repository: https://github.com/mxsena/pattern-programs
let pattern = ""

for (let i = 5; i >= 1; i--) {
    for (let j = 5; j >= 1; j--) {
        pattern += j + " "
    }
    pattern += "\n"
}
console.log(pattern)

File: Solutions/C/pattern005.c
// Pattern Programs n.005 [C]
// Github Repository: https://github.com/mxsena/pattern-programs
#include <stdio.h>

int main(void)
{
    int i, j;

    for (i = 5; i >= 1; i--)
    {
        for (j = 5; j >= 1; j--)
        {
            printf("%d ", j);
        }
        printf("\n");
    }
    return 0;
}